dts/bindings/pinctrl/atmel,sam-pinctrl.yaml

```diff
@@ -20,18 +20,3 @@ child-binding:
     properties:
       "atmel,pins":
         type: phandle-array
-
-      bias-pull-up:
-        required: false
-        type: boolean
-        description: pull up the pin
-
-      bias-pull-down:
-        required: false
-        type: boolean
-        description: pull down the pin
-
-      drive-open-drain:
-        required: false
-        type: boolean
-        description: drive with open drain
```

Comment:
It seems that somehow we need inform user about what props are available.

1) Is there any consensus how to do it?
2) I noted that **slew-rate** type on ST's files are string and at https://www.kernel.org/doc/Documentation/devicetree/bindings/pinctrl/pincfg-node.yaml defines as uint32:
```
  slew-rate:
    $ref: /schemas/types.yaml#/definitions/uint32
    description: set the slew rate

```
https://github.com/zephyrproject-rtos/zephyr/blob/f219a982ffda7496549fab82ae864afffb1303b9/dts/bindings/pinctrl/st%2Cstm32-pinctrl.yaml#L76-L87
It will be better if you already have the final solution on master, so I can rebase and apply [1].

CC: @erwango , @galak